build/ci/.azure-devops-winui-convert.yml

```diff
@@ -24,12 +24,12 @@ jobs:
   steps:
   - checkout: self
     clean: true
+    fetchTags: false
+    fetchDepth: 1
```

Comment:
Versioning of sources may happen here as well, let's keep the full clone+tags

Reply:
@jeromelaban The jobs calling `.azure-devops-winui-convert.yml` should be doing the versioning if required. I think this job only affects source files. All binaries produced during this job are discarded in `Cleanup sources before publish` step.

Reply:
In this context, the versions we're looking for are the ones provided by informational version and related attributes. It may not be needed anymore to modify sources as it's msbuild that includes them. We just need to make sure that the output packages binaries continue to have these versions set.